wagtail/admin/views/account.py

```diff
@@ -1,4 +1,5 @@
 from collections import OrderedDict
+from functools import cached_property
```

Comment:
Just wanted to note that we've been importing this from `django.utils.functional` in all other places, because `functools.cached_property` wasn't available until Python 3.8. Though now that we've dropped Python 3.7 in #10676, this might be OK. Just wondering what we should use in the future (and whether we should refactor the existing ones), I'll raise this in the core team meeting tomorrow.

Reply:
Thanks. Sorry I didn't dig into that further as the tests all passed. I think it's worth having one place that this is imported from consistently, Python-land makes sense if it's going to work well.